MaskRCNN/model/generalized_rcnn.py

```diff
@@ -293,8 +293,8 @@ def roi_heads(self, image, features, proposal_boxes, targets, inputs):
                 per_image_fg_labels = []
                 for i in range(cfg.TRAIN.BATCH_SIZE_PER_GPU):
 
-                    single_image_gt_count = prepadding_gt_counts[i] # 1-D Num_GT_boxes_current_image
-                    single_image_gt_masks = gt_masks[i, :single_image_gt_count, :, :] # Num_GT_boxes_current_image x H_gtmask x W_gtmask (maybe? might have length 1 dim at beginning)
+                    single_image_gt_count = prepadding_gt_counts[i] # 1-D Num_gt_boxes_current_image
+                    single_image_gt_masks = gt_masks[i, :single_image_gt_count, :, :] # Num_gt_boxes_current_image x H_gtmask x W_gtmask (maybe? might have length 1 dim at beginning)
```

Comment:
Remove the comment in parenthesis. It will not have a dimension of 1 in the beginning.